text/0813-template-layering.md

```diff
@@ -0,0 +1,857 @@
+---
+Stage: Accepted
+Start Date: 2022-04-16
+Release Date: Unreleased
+Release Versions:
+  ember-source: vX.Y.Z
+  ember-data: vX.Y.Z
+Relevant Team(s): Ember.js
+RFC PR: https://github.com/emberjs/rfcs/pull/813
+---
+
+# Layering and Desugaring for First-Class Component Templates
+
+## Summary
+
+This RFC propose to introduce a `template()` function as an alternative for the
+`<template>` syntax extension defined in [RFC #779][rfc-779] using standard
+JavaScript, for use in environments where the custom syntax extension is not
+available, and also as a publishing format for addons on NPM:
+
+[rfc-779]: ./0779-first-class-component-templates.md
+
+```js
+import { template } from "@ember/template-compilation";
+import Hello from "my-app/components/hello";
+
+// <template><Hello></template> becomes...
```

Comment:
Should't it be `<template><Hello /></template>` ?
If so, the typo is in the next block also.

Reply:
yup